emails/views.py

```diff
@@ -96,11 +98,12 @@ def _get_relay_address_from_id(request_data, user_profile):
             user=user_profile.user
         )
         return relay_address
-    except RelayAddress.DoesNotExist as e:
+    except RelayAddress.DoesNotExist:
         return HttpResponse("Address does not exist")
 
 
 def _index_PUT(request_data, user_profile):
+    incr_if_enabled('emails_index_put', 1)
```

Comment:
Same question here, should this increment be closer to the successful `JsonResponse`?

Reply:
Hmm ... I was thinking these would record counters of just the requests made - not just the successful ones.

Reply:
I think we can go with treating these counters as overall "raw" counts of operations performed. We have other ways to count successful email calls (DB queries) and failed email calls (sentry errors).